packages/ui/src/lib/adapters/zkitter-god-mode/index.ts

```diff
@@ -71,8 +70,8 @@ export class ZkitterAdapterGodMode extends ZkitterAdapter {
 
 		await this.zkitter.services.pubsub.publish(post, proof)
 
-		const repProof = await this.genRepProof(getGlobalAnonymousFeed(), 5)
-		console.log(repProof)
+		// const repProof = await this.genRepProof(getGlobalAnonymousFeed(), 5)
```

Comment:
must comment this out because you won't be able to generate a rep proof with minimum rep of 5 (since new user has no rep)